nikl_m.py

```diff
@@ -72,7 +73,11 @@ def _process_utterance(out_dir, index, speaker_id, wav_path, text):
         wav = wav / np.abs(wav).max() * hparams.rescaling_max
 
     # Compute the linear-scale spectrogram from the wav:
-    spectrogram = audio.spectrogram(wav).astype(np.float32)
+    try:
+        spectrogram = audio.spectrogram(wav).astype(np.float32)
+    except:
+        print(wav_path)
+        print(wav)
     n_frames = spectrogram.shape[1]
```

Comment:
This is just for debugging?

Reply:
O.O thought I removed it already.
Will remove it.